CP1404_practicals/prac_05/emails.py

```diff
@@ -28,4 +25,11 @@ def get_name_from_email(user_email):
     return name
 
 
+# align the emails so they are in one nice column
+def nice_display(email_information):
```

Comment:
This function name may be improved by making it sound like a verb phase (e.g. display_name_and_email). This can be tested by seeing if it fits into the sentence: "This function will...".